app/data/census_household_cy.json

```diff
@@ -0,0 +1,4786 @@
+{
+    "mime_type": "application/json/ons/eq", 
+    "questionnaire_id": "", 
+    "schema_version": "0.0.1", 
+    "data_version": "0.0.2", 
+    "survey_id": "census", 
+    "title": "Census England Household Schema", 
+    "description": "Census England Household Schema", 
+    "theme": "census", 
+    "navigation": true, 
+    "eq_id": "census", 
+    "groups": [
+        {
+            "id": "who-lives-here", 
+            "title": "Pwy sy’n byw yma?", 
+            "completed_id": "who-lives-here-completed", 
+            "highlight_when": [
+                "who-lives-here-relationship", 
+                "who-lives-here-interstitial"
+            ], 
+            "blocks": [
+                {
+                    "type": "questionnaire", 
+                    "id": "permanent-or-family-home", 
+                    "title": "Who lives here?", 
```

Comment:
We don't have translations for a lot of the block titles (we only have translations where the block title is the same as another string somewhere else), so I thought it was better to be consistent and not translate any of them.